src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -429,6 +515,339 @@
 				</combine>
 			</entity>
 		</macro>
+
+		<macro name="setLabelVariableForPersonsWithoutGNDId">
+			<combine name="@preferredNameForThePerson$[field]" value="${a}"
+				flushWith="$[field][-amn][12].9|$[field][bcefp][1234].9|$[field][-aemn][12].[pa]|$[field][bcf]1.[pa]">
+				<data source="$[field][-aemn][12].[pa]|$[field][bcf]1.[pa]" name="a">
+					<replace pattern="&gt;&gt;" with=""/>
+					<replace pattern="&lt;&lt;" with=""/>
+				</data>
+				<data
+					source="$[field][-amn][12].9|$[field][bcefp][1234].9|$[field][-aemn][12].[pa]|$[field][bcf]1.[pa]"/>
+			</combine>
+		</macro>
```

Comment:
Are the subfields "9" necessary? At least the macro is called "setLabelVariableForPersons**WithoutGNDId**". 

Is the second data tag necessary? The first one should do it to, shouldn't it?